layouts/default.vue

```diff
@@ -9,27 +9,17 @@
         <Nuxt />
 
         <!-- Next page button -->
-        <next-page v-if="getNextPage()!==''" />
```

Comment:
I believe we still need a condition to prevent rendering `next-page` component when we get to the download page. Currently you can see a small gray button displayed.

```suggestion
        <next-page v-if="$route.name !== 'download'"/>
```